src/database/sqlite/sqlite_metadata.c

```diff
@@ -1,6 +1,94 @@
 // SPDX-License-Identifier: GPL-3.0-or-later
 
 #include "sqlite_metadata.h"
+#include "sqlite3recover.h"
+//#include "sqlite_db_migration.h"
+
+#define DB_METADATA_VERSION 16
+
+const char *database_config[] = {
+    "CREATE TABLE IF NOT EXISTS host(host_id BLOB PRIMARY KEY, hostname TEXT NOT NULL, "
+    "registry_hostname TEXT NOT NULL default 'unknown', update_every INT NOT NULL default 1, "
+    "os TEXT NOT NULL default 'unknown', timezone TEXT NOT NULL default 'unknown', tags TEXT NOT NULL default '',"
+    "hops INT NOT NULL DEFAULT 0,"
+    "memory_mode INT DEFAULT 0, abbrev_timezone TEXT DEFAULT '', utc_offset INT NOT NULL DEFAULT 0,"
+    "program_name TEXT NOT NULL DEFAULT 'unknown', program_version TEXT NOT NULL DEFAULT 'unknown', "
+    "entries INT NOT NULL DEFAULT 0,"
+    "health_enabled INT NOT NULL DEFAULT 0, last_connected INT NOT NULL DEFAULT 0)",
+
+    "CREATE TABLE IF NOT EXISTS chart(chart_id blob PRIMARY KEY, host_id blob, type text, id text, name text, "
```

Comment:
Considering that first table (` host`)  has default values for all fields, I suggest we also apply this pattern for others.
Considering that we are reorganizing, can we consider this PR the first step to reorganize our tables? For example, creating some relationship `N <-> M`? :laughing: (We already talked about this.